scripts/examples/ota_test.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+PASSCODE=${1:-20202021}
+ID=${2:-42}
+DISCRIMINATOR=${3:-5560}
+
+pkill chip-ota-provider-app
+pkill chip-ota-requestor-app
+
+scripts/examples/gn_build_example.sh examples/chip-tool out/
+
+touch my-firmware.bin
+
+./src/app/ota_image_tool.py create -v 0xDEAD -p 0xBEEF -vn 1 -vs "1.0" -da sha256 my-firmware.bin my-firmware.ota
+
+if [ ! -f "my-firmware.ota" ]; then
+    exit 1
+fi
+
+./out/ota_provider_debug/chip-ota-provider-app -f my-firmware.ota | tee /tmp/ota/provider-log.txt &
+provider_pid=$!
+
+if [ ! -f "/tmp/ota/provider-log.txt" ]; then
+    exit 1
+fi
+
+echo "Commissioning Provider "
+
+./out/chip-tool pairing onnetwork 1 "$PASSCODE" | tee /tmp/ota/chip-tool-commission-provider.txt
+if grep "Device commissioning completed with success" /tmp/ota/chip-tool-commission-provider.txt; then
+    echo Provider Commissioned
+else
+    echo Provider not commissioned properly
+    exit 1
+fi
+
+rm /tmp/chip_kvs
+
+stdbuf -o0 ./out/ota_requestor_debug/chip-ota-requestor-app -u "$DISCRIMINATOR" -d "$ID" | tee /tmp/ota/requestor-log.txt &
```

Comment:
The `-u` is for UDP port and `-d` is for discriminator. I think the values are supplied incorrectly here (ID for the -d option).

Reply:
nit: `ID` should probably be more aptly named